reward/reward_distributor.go

```diff
@@ -75,14 +75,16 @@ type rewardConfig struct {
 }
 
 type RewardSpec struct {
-	Minted   *big.Int                    `json:"minted"`   // the amount newly minted
-	TotalFee *big.Int                    `json:"totalFee"` // total tx fee spent
-	BurntFee *big.Int                    `json:"burntFee"` // the amount burnt
-	Proposer *big.Int                    `json:"proposer"` // the amount allocated to the block proposer
-	Stakers  *big.Int                    `json:"stakers"`  // total amount allocated to stakers
-	Kgf      *big.Int                    `json:"kgf"`      // the amount allocated to KGF
-	Kir      *big.Int                    `json:"kir"`      // the amount allocated to KIR
-	Rewards  map[common.Address]*big.Int `json:"rewards"`  // mapping from reward recipient to amounts
+	Minted   *big.Int `json:"minted"`   // the amount newly minted
+	TotalFee *big.Int `json:"totalFee"` // total tx fee spent
+	BurntFee *big.Int `json:"burntFee"` // the amount burnt
+	Proposer *big.Int `json:"proposer"` // the amount allocated to the block proposer
+	Stakers  *big.Int `json:"stakers"`  // total amount allocated to stakers
+	// TODO-klaytn-fund: change kff json tag to kff
+	KFF *big.Int `json:"kgf"` // the amount allocated to KGF
+	// TODO-klaytn-fund: change kcf json tag to kcf
+	KCF     *big.Int                    `json:"kcf"`     // the amount allocated to KIR
```

Comment:
According to the PR description, shouldn't the json tag remain "kir" in this PR? and "KIR" in the comment should be updated to "KCF"?

Reply:
Nice catch! Yes, you are right. I reverted it